src/IssueViz.Security/OpenInIDE/Http/OwinPipelineProcessor.cs

```diff
@@ -58,10 +61,29 @@ public OwinPipelineProcessor([ImportMany]IEnumerable<IOwinPathRequestHandler> pa
         /// <summary>
         /// Called for each low-level HTTP request received by the underlying HTTPListener
         /// </summary>
-        public Task ProcessRequest(IDictionary<string, object> environment)
+        public async Task ProcessRequest(IDictionary<string, object> environment)
```

Comment:
Simple implementation to handle urecognized messages and errors:
* all requests are logged
* unrecognised requests return a 200 status code
* exceptions are logged and rethrown. We expect the calling component turn this into a 500 error code (and there's an integration test to check this is the case)